Lib/fontmake/font_project.py

```diff
@@ -236,7 +238,9 @@ def _font_name(self, ufo):
     def _output_dir(self, ext, is_instance=False):
         """Generate an output directory."""
 
-        dir_prefix = 'instance_' if is_instance else 'master_'
+        dir_prefix = os.path.join(
+            'out_compatible' if self.compatible else 'out',
```

Comment:
For ufo and otf, interpolatable doesn't make sense.  They are always interpolatable.  It only matters for TTF.  And for instances it doesn't make any sense either.

It's one configuration that I need: "TrueType outlines, interpolatable, overlaps-retained, masters".  Just add a config for that particular case?